docs/section5/Rule5-1.md

```diff
@@ -0,0 +1,64 @@
+                - Check if surface is above-grade wall: `if get_opaque_surface_type(surface) == "ABOVE-GRADE WALL":`   
+                    - Get the azimuth: `surface_azimuth = surface.azimuth`  
+                    - Check if the azimuth is not in a key in the dictionary of azimuths and fen area, if it is not then add it: `if surface_azimuth not in azimuth_fen_area_dict:`   
+                        - Add the unique azimuth to the dictionary as a key and set initial fen area to 0: `azimuth_fen_area_dict[surface_azimuth]= 0`  
+                    - Reset the total surface fenestration area variable to 0: `total_surface_fenestration_area = 0`  
+                    - For each subsurface associated with the surface: `for sub_surface in surface.subsurfaces:`   
+                        - Check if subsurface is door: `if sub_surface.classification == "DOOR":`
+                            - If glazed area in door is more than 50% of the total door area, add door area to total_surface_fenestration_area: `if sub_surface.glazed_area > subsurface.opaque_area: total_surface_fenestration_area += sub_surface.glazed_area + sub_surface.opaque_area`
+                        - Else, subsurface is not door, add total area to total_surface_fenestration_area (because we checked that the surface is an above grade wall we can assume that the subsurface classification is not and could not be a skylight so no need to check this): `total_surface_fenestration_area += subsurface.glazed_area + subsurface.opaque_area`      
+                    - Add the total_surface_fenestration_area summed for the surface to the total fen area associated with the azimuth: `azimuth_fen_area_dict[surface_azimuth] += total_surface_fenestration_area`    
+
+Check if the area differs by 5 percent or more.
+- Get the max fen area: `max_fen_area = azimuth_fen_area_dict[max(azimuth_fen_area_dict, key=azimuth_fen_area_dict.get)]`  
+- Get the min fen area: `min_fen_area = azimuth_fen_area_dict[min(azimuth_fen_area_dict, key=azimuth_fen_area_dict.get)]`  
+- Calculate the % difference, take the maximum calculated: `percent_difference = max(abs(max_fen_area- min_fen_area)/max_fen_area,abs(min_fen_area- max_fen_area)/min_fen_area)` 
+- Check if the % difference is 5% or more, if it is then set rotation_expected boolean to TRUE: `if percent_difference >= 5%: rotation_expected = TRUE`  
+- Else, set rotation_expected to FALSE: `else: rotation_expected = FALSE`  
+
+- Set counter variable to 0 (counts the number of output instances): `counter =0`
+- Get the number of ruleset_model_descriptions: `number_of_RMIs = len(ASHRAE229.ruleset_model_descriptions)`  
+- Check that there is an output_instance associated with each RMI and add to the counter variable for each one `For RMI in ASHRAE229.ruleset_model_descriptions:`  
+    - Check if there is an output_instance associated with the RMI: `if RMI.output.Output2019ASHRAE901.output_instance != Null: counter += 1 `
```

Comment:
I am not sure what output values we want to check here? output is just an object, checking this is equivalent to checking it is an non-empty object.

Reply:
@weilixu we are just checking that output results were provided for the rmd. Is there another way to do this?

Reply:
Got it. I want to confirm that is the intention. Thanks!